src/pages/docs/FAQs/mobile-apps/how-to-resolve-app-launch-issues.md

```
@@ -0,0 +1,45 @@
+2. In the **Test Case Explorer** section, expand the required **Feature** and **Scenario** and select the required test case.
+   ![Select a test case](https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/faq/IPA_2.png)
+
+3. In the **Test Case Details** page, click **Run**.
+   ![Run](https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/faq/IPA_3.png)
+
+4. In the **Ad-Hoc Run** overlay, expand the **Desired Capabilities** field and specify the following desired capability:
+
+   |Key|Data type|Value|
+   |---|---|---|
+   |resignApp|String|false|
+   ![desired capability](https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/faq/IPA_4.png)
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)